### PR TITLE
fix(subscription): chat active on subscribe

### PR DESCRIPTION
**Pull request by Jekrimo**
**What this PR does** 📖
Block  message send until subscription happens - add message to alert to blocked sending

AP-2050

### CODE REVIEW

**Review by aewing — APPROVED** [review]

**Comment by aewing** [comment]
Nice!

**Comment by aewing** [comment]
To test this we just want to make sure you can't send a message in a conversation until the subscription is active (messages sent always show up on the other side and you are prevented from sending briefly during initial load)

**Comment by stavares843** [comment]
thanks 🔨

File: package.json [hunk 1]
```diff
@@ -33,7 +33,6 @@
     "@nuxtjs/style-resources": "^1.2.1",
     "@nuxtjs/toast": "^3.3.1",
     "@project-serum/anchor": "0.18.2",
-    "@satellite-im/iridium": "git+https://git@github.com/Satellite-im/iridium.git#main",
```
Comment (stavares843) [inline package.json:36]: is this on purpose? 🔨

**Pull request merged**